core/src/test/java/org/elasticsearch/plugins/PluginManagerUnitTests.java

```diff
@@ -63,8 +62,8 @@ public void testThatConfigDirectoryCanBeOutsideOfElasticsearchHomeDirectory() th
         Environment environment = new Environment(settings);
 
         PluginManager.PluginHandle pluginHandle = new PluginManager.PluginHandle(pluginName, "version", "user");
-        String configDirPath = Files.simplifyPath(pluginHandle.configDir(environment).normalize().toString());
-        String expectedDirPath = Files.simplifyPath(genericConfigFolder.resolve(pluginName).normalize().toString());
+        String configDirPath = pluginHandle.configDir(environment).normalize().toString();
+        String expectedDirPath = genericConfigFolder.resolve(pluginName).normalize().toString();
```

Comment:
I think at the very start we didn't call `normalize`, after we added normalize we didn't remove `simplifyPath`. I'm not sure if this was on purpose. 

I did a quick test around this, check this out   https://github.com/xuzha/guava/commit/d2865f81eaca0257c002362364d9439877ab151a. From google's test cases, looks like only difference is `normalize` will just remove the single `.`. 

I think this is OK for this unit test.